gstd/gstd_tcp.c

```diff
@@ -331,6 +361,10 @@ gstd_tcp_init_get_option_group (GstdIpc * base, GOptionGroup ** group)
     {"enable-tcp-protocol", 't', 0, G_OPTION_ARG_NONE, &base->enabled,
         "Enable attach the server through given TCP ports ", NULL}
     ,
+    {"host", 'h', 0, G_OPTION_ARG_STRING, &self->host,
```

Comment:
Can we use `address` and `a` instead of `host` and `h`. I don't like it when `h` gets overridden.